contracts/tokens/OZStorage.sol

```diff
@@ -2,17 +2,15 @@ pragma solidity ^0.5.0;
 
 contract OZStorage {
 
-    mapping (address => uint256) private _balances;
+    mapping (address => uint256) public _balances;
```

Comment:
public variables generate a getter with the same name that takes some code size. We should instead define these variables as internal.

Reply:
But we'll lose the etherscan.io read contract tab with proxied tokens, so having basic information available seems useful in etherscan.